pages/posts/portal-bridge-latest-guide.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to run Portal Bridge on history latest mode Windows/Linux guide
+date: 2024/2/15
+description: Explaining how to run a Portal Bridge in history latest mode.
+tag: portal bridge, Ethereum, archive
+author: Kolby ML (Moroz Liebl)
+---
+
+# What is a Portal Bridge?
+A Portal Bridge is a application which takes Ethereum mainnet data from a source and provides it to a Portal client to inject into Portal sub-networks.
+
+# What are we doing in this guide?
+We will be deploying a Portal Bridge using the free tier of infura. The bridge will be running the History latest mode, which gossips the latest Ethereum block data onto the Portal History network. The Portal History network is a Portal sub-network which stores canonical Ethereum mainnet headers, block bodies, and receipts.
+
+# Guide
+<details><summary>Step 1: Get infura endpoint + api key url</summary>
+
+**Step 1:** goto https://www.infura.io/ and click get started
+![Image of Infura](../../public/images/portal-bridge-latest-guide/image-of-infura.png)
+**Step 2:** Create an account or login
+![Create an account page](../../public/images/portal-bridge-latest-guide/create-an-account-page.png)
+**Step 3:** Create a api key
+![Create a api key](../../public/images/portal-bridge-latest-guide/create-a-api-key.png)
+**Step 4:** Click mainnet then save changes
+![Click mainnet then save changes](../../public/images/portal-bridge-latest-guide/click-mainnet-then-save-changes.png)
+**Step 5:** Copy the infura endpoint and save it for later
+![Copy the infura endpoint and save it for later](../../public/images/portal-bridge-latest-guide/copy-the-infura-endpoint-and-save-it-for-later.png)
+</details>
+<details><summary>Step 2: Install depends Windows!</summary>
```

Comment:
`Install depends Windows!` is hard to understand... This can only be run with Windows?